client/state/comments/models/comment.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Internal dependencies
+ */
+import { Author } from './comment-author'; // eslint-disable-line no-unused-vars
+import { CommentInfo } from './comment-info'; // eslint-disable-line no-unused-vars
+
+export function Comment( constructor ) {
+	Object.defineProperty( this, 'constructor', {
+		value: constructor || this
+	} );
+}
+
+export function LoadingComment( siteId, commentId ) {
```

Comment:
The idea I am trying to convey is that "we're aware that t his comment exists but we don't yet have its content" so in this sense it's loading. This means that we're leaving the responsibility of loading it for Calypso.

The way I see this working is by grabbing the list of comments and creating `LoadingComment` objects from that list, then as data comes in they turn into `LoadedComment` objects.